msgs/thrift/WearableActuators.thrift

```diff
@@ -22,8 +22,16 @@ struct ActuatorInfo {
 // Actuator Command data type
 // ==========================
 
+// struct WearableActuatorCommand {
+//  1: ActuatorInfo info;
+//  2: double forceValue;
+//  3: double vibroTactileValue;
+//  4: double duration;
+// }
+
 struct WearableActuatorCommand {
   1: ActuatorInfo info;
-  2: double value;
-  3: double duration;
+  2: list<double> forceValue;
+  3: list<double> vibroTactileValue;
+  4: list<double> duration;
```

Comment:
The use of lists inside the message is not super clean, as you need to enforce in some way that each list has the same dimension (I guess).

What about defining a new message that instead contains a list of ``WearableActuatorCommand``?

I am thinking something like this:
```thrift
struct WearableActuatorCommand {
  1: ActuatorInfo info;
  2: double value;
  3: double duration;
}

struct WearableActuatorCommandList {
  1: list<WearableActuatorCommand> commands;
}
```

Reply:
Same as below, in this case, I think it is more readable since vibro and force are in separate vectors.